src/Utils/QtiMarshallerUtil.php

```diff
@@ -55,8 +58,26 @@ public static function marshallValidCollection(QtiComponentCollection $collectio
     public static function marshallCollection(QtiComponentCollection $collection)
     {
         $results = [];
+        $html = '';
         foreach ($collection as $component) {
-            $results[] = self::marshall($component);
+            if ($component instanceof ObjectElement) {
+                $class = new \ReflectionClass(get_class($component));
+                if (property_exists($component, 'data')) {
+                    $property = $class->getProperty('data');
+                    $property->setAccessible(true);
+                    $learnosityServiceObject = ConvertToLearnosityService::getInstance();
+                    $inputPath = $learnosityServiceObject->getInputpath();
+                    $file = $inputPath . '/' . $property->getValue($component);
+                    if (file_exists($file)) {
+                        $html = HtmlExtractorUtil::getHtmlData(realpath($file));
+                        $results[] = $html;
+                    } else {
+                        echo 'File not found: ' . $file . PHP_EOL;
+                    }
+                }
+            } else if(!($component instanceof FeedbackInline)) {
```

Comment:
Tiny one - would be preferable if you can change this `else if` into an `elseif` (No functional difference, more a coding style thing. `else if` is actually interpreted as an `else` statement with an `if` block inside it, rather than a single distinct branch of the `if` block. It's a PSR-2 recommendation, though not a requirement so I'm not too fussed).

Reply:
Done